lib/app/timer.ex

```diff
@@ -1,117 +1,98 @@
 defmodule App.Timer do
   use Ecto.Schema
   import Ecto.Changeset
-  import Ecto.Query, warn: false
+  # import Ecto.Query
   alias App.Repo
   alias __MODULE__
+  require Logger
 
   schema "timers" do
-    field :end, :naive_datetime
-    field :start, :naive_datetime
     field :item_id, :id
     field :person_id, :id
+    field :start, :naive_datetime
+    field :stop, :naive_datetime
```

Comment:
`start` and `stop` makes a lot more sense to me _alphabetically_ and does not confuse `Elixir`. 👍